hackathon_site/dashboard/frontend/src/assets/images/icons/BackArrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="16" viewBox="0 0 24 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Use a MUI icon instead:
```javascript
import ArrowBackIcon from '@material-ui/icons/ArrowBack';
```

Reply:
sad okay :'( Graham convinced me to use toe MUI one